src/components/Modal/Modal.stories.tsx

```diff
@@ -46,14 +46,17 @@ const getChildren = (
     >
       {showStepper && <Modal.Stepper activeStep={2} totalSteps={5} />}
       <ButtonGroup className={styles['footer__button-group']}>
-        <Button
-          onClick={
-            () => {} /* eslint-disable-line @typescript-eslint/no-empty-function */
-          }
-          status="neutral"
-        >
-          Button 1
-        </Button>
+        {/* This has to be manually tested since Tooltip tests are flaky in Chromatic */}
+        <Tooltip text="Tooltip should spawn on top of modal">
```

Comment:
ooh out of scope of this ticket but just had a thought -- if instead of passing `visible: true` to the tooltip stories, we used a play function to focus on them, would that help the flakiness?

Reply:
I initially tried using play function to hover and capture the snap with chromatic, but I wasn't having a successful time. Could spend more time later on it tho